packages/compass-web/scripts/electron-proxy.js

```diff
@@ -112,6 +129,23 @@ class AtlasCloudAuthenticator {
     return new URL(url, 'http://localhost').pathname.replace('/v2/', '');
   }
 
+  async #fetch(path, init) {
```

Comment:
I keep forgetting that private properties are even a thing for literally years at a time 😆